### PR TITLE
Created a README for Instagram Clone

### PR DESCRIPTION
**Pull request by afred-khan**

### CODE REVIEW

**Review by afred-khan — COMMENTED** [review]
I have added the README file

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Instagram Clone
+
+Instagram clone built with React, Redux, Node.js, Express, Passport.js & MongoDB.
```
Comment (stephin007) [inline README.md:3]: Remove redux and passportjs

File: README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Features
+
+- Sign up for an account and edit avatar/bio/name later on
```
Comment (stephin007) [inline README.md:7]: Sign up is currently has only name and email, so  remove avatar

File: README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- Upload images with captions and option to edit/delete them
+- Comment on posts and option to edit/delete them
+- Like/unlike posts
+- Follow/unfollow users
+- Feed consists of posts from users you follow and the 'Explore' page has posts from every user
```
Comment (stephin007) [inline README.md:12]: NO need of this line!

File: README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+## Features to add
```
Comment (stephin007) [inline README.md:14]: Remove the points below under this section

File: README.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+- Search option for tags and location
+- Bookmark posts
+- Modals that display likes (on a post), and followers/following (on profile)
+
+## Running Locally
+
+The following instructions should get this project running on your machine.  
+
+### Prerequisites
+1. Install [MongoDB](https://www.mongodb.com/)
```
Comment (stephin007) [inline README.md:28]: You need not install mongo, you have to connect your mongo db with your project by creating a  free mongo db account and adding the database user name and databse password in the keys.js.example file

File: README.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+- Bookmark posts
+- Modals that display likes (on a post), and followers/following (on profile)
+
+## Running Locally
+
+The following instructions should get this project running on your machine.  
+
+### Prerequisites
+1. Install [MongoDB](https://www.mongodb.com/)  
+2. Create a [Cloudinary](https://cloudinary.com/) account (need an API key and secret)
```
Comment (stephin007) [inline README.md:29]: Remove this line!

File: README.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+### Installing
+```
+git clone https://github.com/101Loop/InstagramClone-BE.git
+cd reactgram
```
Comment (stephin007) [inline README.md:34]: Remove this line

File: README.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+npm install
+cd client
+npm install
+```
```
Comment (stephin007) [inline README.md:38]: Remove line 36 and 37

File: README.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+Create a .env file in the root of the project and add the following:
```
Comment (stephin007) [inline README.md:40]: There is already an keys.js.env 
- you just have to copy that file and rename it to keys.js and then add databaseusername and Password.
- secondly, you have to add a JWT token, which shlould be any random alphabets atleast 14 characters

File: README.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+API_SECRET=<secret>
+```
+
+Run `mongod` in another terminal and `npm run dev` in the terminal with the project.
+
+## Tests
+
+```
+npm test
+```
```
Comment (stephin007) [inline README.md:54]: Remove lines 42-54 (no need as of now)

**Review by stephin007 — CHANGES_REQUESTED** [review]
Please look into the comments and update the readme accordingly

**Review by afred-khan — COMMENTED** [review]
Done with the changes

**Review by stephin007 — CHANGES_REQUESTED** [review]
You still haven't done the changes! you have just marked as resolved!
Because i am not able to see any changes

**Review by afred-khan — COMMENTED** [review]
Review it!

**Review by stephin007 — APPROVED** [review]
LGTM

**Pull request merged**